From: Richard B Winters <rik@mmogp.com>
Date: Wed, 6 May 2015 01:40:59 -0400
Subject: Remove require on rubygems in pcsd ssl.rb

  - Remove the require on rubygems from /usr/share/pcsd/ssl.rb 
    as dependencies are handled via debian

Change-Id: I0c2686f52aab375ec67a7686c911c79e776f8fe8
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/ssl.rb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pcsd/ssl.rb b/pcsd/ssl.rb
index 1001755..6535b88 100644
--- a/pcsd/ssl.rb
+++ b/pcsd/ssl.rb
@@ -1,4 +1,3 @@
-require 'rubygems'
 require 'webrick'
 require 'webrick/https'
 require 'openssl'
